src/Sylius/Bundle/ApiBundle/DependencyInjection/Configuration.php

```diff
@@ -38,6 +38,29 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->defaultValue('media/image')
                 ->end()
             ->end()
+            ->children()
+                ->arrayNode('filter_eager_loading_extension')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('restricted_operations')
+                            ->useAttributeAsKey('name')
+                            ->arrayPrototype()
+                                ->children()
+                                    ->arrayNode('resources')
+                                        ->useAttributeAsKey('name')
+                                        ->arrayPrototype()
+                                            ->canBeDisabled()
+                                            ->children()
+                                                ->booleanNode('enabled')->defaultTrue()->end()
```

Comment:
If I understands this tree correctly we have following configs possible:

```yml
sylius_api:
    filter_eager_loading_extension:
        restricted_operations:
            shop_get:
                resources:
                    '%sylius.model.product.class%': ~
```
and equivalent 
```yml
sylius_api:
    filter_eager_loading_extension:
        restricted_operations:
            shop_get:
                resources:
                    '%sylius.model.product.class%': true
```
and equivalent 
```yml
sylius_api:
    filter_eager_loading_extension:
        restricted_operations:
            shop_get:
                resources:
                    '%sylius.model.product.class%': 
                        enabled: true
```

and on another spectrum:
```yml
sylius_api:
    filter_eager_loading_extension:
        restricted_operations:
            shop_get:
                resources:
                    '%sylius.model.product.class%': false
```
and equivalent 
```yml
sylius_api:
    filter_eager_loading_extension:
        restricted_operations:
            shop_get:
                resources:
                    '%sylius.model.product.class%': 
                        enabled: false
```

but we have an extension test only for one of them. Can you add a configuration test for other possibilities? Please omit the ones, that I misunderstood